tests/integration/conftest.py

```diff
@@ -82,6 +83,22 @@ def test_extractor():
         pass
 
 
+
+@pytest.fixture(scope="function")
+def test_data_modeling_replicator():
+    stop_event = CancellationToken()
+    replicator = DataModelingReplicator(metrics=safe_get(Metrics), stop_event=stop_event)
+    replicator._initial_load_config(override_path=os.environ["TEST_CONFIG_PATH"])
+    replicator.cognite_client = replicator.config.cognite.get_cognite_client(replicator.name)
+    replicator._load_state_store()
+    replicator.logger = Mock()
+    yield replicator
+    try:
+        os.remove("states.json")
```

Comment:
There is no longer local state store "states.json", the test_config.yaml uses a raw state store now. The remote_state_store fixture handles the setup and cleanup of this, so you can just refer to this fixture instead.